src/app/manage/manage.component.ts

```diff
@@ -10,7 +11,7 @@ import { UserStructure } from '../user-structure.model';
 })
 export class ManageComponent implements OnInit {
 
-  users: UserStructure[];
+  users: Observable<UserStructure[]>;
```

Comment:
See my comment for naming observables above and also change the model name to `User`